modules/astraoneBidAdapter.md

```diff
@@ -109,90 +112,91 @@ var adUnits = [{
-					sizes: [1, 1]
-				}
-			},
-			bids: [{
-				bidder: "astraone",
-				params: {
-					placement: "inImage",
-					placeId: "5af45ad34d506ee7acad0c26",
-					imageUrl: "https://creative.astraone.io/files/default_image-1-600x400.jpg"
-				}
-			}]
-		}];
-
-		var googletag = googletag || {};
-		googletag.cmd = googletag.cmd || [];
-
-		googletag.cmd.push(() => {
-			googletag.pubads().disableInitialLoad();
-		});
-
-		pbjs.que.push(() => {
-			pbjs.addAdUnits(adUnits);
-			pbjs.requestBids({ bidsBackHandler: sendAdServerRequest });
-		});
-
-		function sendAdServerRequest() {
-			googletag.cmd.push(() => {
-				pbjs.que.push(() => {
-					pbjs.setTargetingForGPTAsync('div-gpt-ad-1574864639578-0');
-					googletag.pubads().refresh();
-				});
-			});
-		}
-
-		googletag.cmd.push(() => {
-			googletag
-				.defineSlot('/21877108735/rtb-pbjs', [1, 1], 'div-gpt-ad-1574864639578-0')
-				.addService(googletag.pubads());
-
-			googletag.pubads().enableSingleRequest();
-			googletag.enableServices();
-		});
-	</script>
+  <meta charset="UTF-8">
+  <title>Prebid.js Banner gbt Example</title>
+  <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
+  <script async src="../../../build/dev/prebid.js"></script>
+  <style>
+    .banner-block {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      top: 0;
+      left: 0;
+      z-index: 1;
+      border: none;
+      overflow: hidden;
+    }
+    .banner-block div {
+      width: 100%;
+      height: 100%;
+    }
+  </style>
+  <script>
+    var sizes = [1, 1];
+    var pbjs = pbjs || {};
+    pbjs.que = pbjs.que || [];
+
+    var adUnits = [{
+      code: 'div-gpt-ad-1574864639578-0',
+      mediaTypes: {
+        banner: {
+          sizes: sizes,
+        }
+      },
+      bids: [{
+        bidder: "astraone",
+        params: {
+          placement: "inImage",
+          placeId: "5f477bf94d506ebe2c4240f3",
+          imageUrl: "https://creative.astraone.io/files/default_image-1-600x400.jpg"
+        }
+      }]
+    }];
+
+    var googletag = googletag || {};
+    googletag.cmd = googletag.cmd || [];
+
+    googletag.cmd.push(() => {
+      googletag.pubads().disableInitialLoad();
+    });
+
+    pbjs.que.push(() => {
+      pbjs.addAdUnits(adUnits);
+      pbjs.requestBids({ bidsBackHandler: sendAdServerRequest });
+    });
+
+    function sendAdServerRequest() {
+      googletag.cmd.push(() => {
+        pbjs.que.push(() => {
+          pbjs.setTargetingForGPTAsync('div-gpt-ad-1574864639578-0');
+          googletag.pubads().refresh();
+        });
+      });
+    }
+
+    googletag.cmd.push(() => {
+      googletag
+        .defineSlot('/21877108735/rtb-pbjs', sizes, 'div-gpt-ad-1574864639578-0')
+        .addService(googletag.pubads());
+
+      googletag.pubads().enableSingleRequest();
+      googletag.enableServices();
+    });
+  </script>
 </head>
 <body>
-	<h2>Prebid.js Banner Ad Unit Test</h2>
+<h2>Prebid.js InImage Banner gbt Test</h2>
```

Comment:
Happened to notice this reading through. Should this be gpt not gbt?

Reply:
Renamed